### PR TITLE
Create LOS_Link Interface and Port PPM Driver

### PR DESCRIPTION
**Pull request by Chrisc110**

### CODE REVIEW

File: LOS_Interface/Inc/config.hpp [hunk 1]
```diff
@@ -2,14 +2,22 @@
...
 #include <cstdint>
+#include "main.h"
+#include "tim.h"
+
 
 /* LOS_Link */
 #define PPM
+const uint8_t MAX_PPM_CHANNELS = 12;
 const uint8_t NUM_RX_CHANNELS = 5;
+const uint16_t PPM_TIMEOUT = 1000;
```
Comment (antholuo) [inline LOS_Interface/Inc/config.hpp:13]: this is in us right?

**Review by antholuo — APPROVED** [review]
ty chris! very epic, is this able to be tested soon on an L5?

File: LOS_Interface/Inc/config.hpp (hunk 1 above)
Comment (Chrisc110) [reply LOS_Interface/Inc/config.hpp:13]: yes

**Comment by Chrisc110** [comment]
> ty chris! very epic, is this able to be tested soon on an L5?

The idea is yes, next week after me guy's mostafa's PRs are merged,

File: LOS_Interface/Inc/config.hpp (hunk 1 above)
Comment (Aidan-B) [reply LOS_Interface/Inc/config.hpp:13]:
```suggestion
const uint16_t PPM_TIMEOUT_US = 1000;
```

**Pull request merged**